src/lib/routes/admin-api/user-admin.ts

```diff
@@ -123,6 +137,50 @@ export default class UserAdminController extends Controller {
             ],
         });
 
+        this.route({
```

Comment:
I would like to have these routes in a separate handler, to avoid to much in this file. Also I think it make sense to move it to the enterprise repo eventually, as service-accounts will be an enterprise feature.